docs/rst/user_manual/wan_configuration.rst

```diff
@@ -68,13 +68,86 @@ These are a list of tips to help choosing whether to use one or the other.
         - Require only server side of the communication |br|
           to have port forwarded from the router.
 
-.. note:
+.. note::
 
     DDS is thought to work over UDP and has its own reliability mechanisms.
     Thus, the |ddsrouter| uses UDP transport by default for every address that has not explicitly specified
     a transport in the configuration file.
 
 
+TLS
+---
+
+|eddsrouter| also supports `TLS over TCP <https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/tcp/tls.html>`_,
+and its configuration can be set per participant for types Local Discovery Server and WAN. Following is a list of the
+accepted entries under the ``tls`` tag:
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Tag
+        - Requiredness
+        - Description
+
+    *   - ``ca``
+        - Mandatory for TLS clients
+        - Path to the CA (Certification- Authority) file.
+
+    *   - ``password``
+        - Optional for TLS servers
+        - Password of the ``private_key_file`` or ``rsa_private_key_file``.
+
+    *   - ``private_key``
+        - Mandatory for TLS servers
+        - Path to the private key certificate file.
+
+    *   - ``cert``
+        - Mandatory for TLS servers
+        - Path to the public certificate chain file.
+
+    *   - ``dh_params``
+        - Mandatory for TLS servers
+        - Path to the Diffie-Hellman parameters file.
+
+Below is an example on how to configure a WAN participant as a TLS server and client:
+
+.. code-block:: yaml
```

Comment:
Move this to a subsection below in the `Examples` Section of this same page.

Reply:
:+1: